perceval/backend.py

```diff
@@ -579,12 +579,11 @@ class BackendCommand:
     """
     BACKEND = None
 
-    def __init__(self, *args):
+    def __init__(self, *args, debug_check=False):
```

Comment:
I think you can rename the parameter to `debug`, which is probably more clear. Also would you mind to add this parameter in the documentation of the class, so people know what to expect? Some examples here: https://github.com/chaoss/grimoirelab-perceval/blob/master/perceval/backends/core/git.py#L65